test/hdt-test.js

```diff
@@ -120,51 +120,47 @@ describe('hdt', function () {
           });
       });
       it('Should get suggestions for literals', function (done) {
-        document.getSuggestions('"a', { position : 'object' },
+        document.findTerms({ prefix: '"a', position : 'object' },
           function (error, suggestions) {
             if (error) return done(error);
             suggestions.should.have.lengthOf(8);
             done();
           });
       });
-      // Skipping, as this needs an hdt-cpp submodule that has this fix:
-      // https://github.com/rdfhdt/hdt-cpp/pull/87
-      it.skip('Should return suggestions on empty match', function (done) {
-        document.getSuggestions('', { position: 'object' },
+      it('Should 0 results on empty match', function (done) {
+        document.findTerms({ prefix: '', position: 'object' },
           function (error, suggestions) {
             if (error) return done(error);
-            suggestions.should.have.lengthOf(100);
+            suggestions.should.have.lengthOf(0);
             done();
           });
       });
-      // Skipping, as this needs an hdt-cpp submodule that has this fix:
-      // https://github.com/rdfhdt/hdt-cpp/pull/87
-      it.skip('Should return suggestions on null match', function (done) {
-        document.getSuggestions(null, { position: 'object' },
+      it('Should 0 results when prefix is not defined', function (done) {
```

Comment:
I wonder if we want an error here perhaps?

Reply:
I was hesitant about this as as well. Perhaps we can simply treat this as a wildcard and return any term in that position

Reply:
Sweet, I'd like that.

Reply:
The failing tests are a bit odd. Locally, with a synced submodule, I get 0 results when autocompleting an empty string. 
In travis the empty string is treated as wildcard though (which is what we happen to want ;) ). Haven't figured out why this differs